src/core/symbology/qgsfillsymbollayer.cpp

```diff
@@ -2576,14 +2576,16 @@ bool QgsLinePatternFillSymbolLayer::hasDataDefinedProperties() const
   return false;
 }
 
-void QgsLinePatternFillSymbolLayer::startFeatureRender( const QgsFeature &, QgsRenderContext & )
+void QgsLinePatternFillSymbolLayer::startFeatureRender( const QgsFeature &, QgsRenderContext &context )
 {
   // deliberately don't pass this on to subsymbol here
```

Comment:
it looks here like the existing comment applies to "installMasks", whereas it's actually referring to the call to startFeatureRender. Can you tweak the comments or add spacing or something to keep this distinction clear?

Reply:
I moved install/remove masks after the comment and completed it a bit to remove ambiguity